matflow_damask/main.py

```diff
@@ -329,6 +330,20 @@ def volume_element_from_microstructure_image(microstructure_image, depth,
     }
     return out
 
+#@func_mapper(task='generate_volume_element', method='extrusion')
+#def volume_element_from_microstructure_image(microstructure_image, depth, image_axes,
+#                                             phase_label, homog_label):
+#    out = {
+#        'volume_element': volume_element_from_2D_microstructure(
+#            microstructure_image,
+#            phase_label,
+#            homog_label,
+#            depth,
+#            image_axes,
+#        )
+#    }
+#    return out
+
```

Comment:
Left over from @mikesmic 's commit `7c5b51ceaf2b0d15f9d50a1268219384053fd712`
"Prep for v0.1.25"